README.md

```diff
@@ -5,6 +5,26 @@
 This project will improve the FPGA code in the [M2 Cell Controller LabVIEW project](https://github.com/lsst-ts/ts_mtm2_cell).
 In that version, the VI named **4-port serial Master Slave.vi**, under `Chassis (cRIO-9038) >> FPGA Target (RIO0, cRIO-9038)` in the Project Explorer, interacts with the hardware inputs and outputs.
 
+## Platform
+
+- Windows 10
+- LabVIEW 2018 SP1 32-bit
+- cRIO-9049 (M2 simulator: m2-crio-simulator.ls.lsst.org)
+
+## Needed Package
+
+- LabVIEW NI Device Drivers (DVD-Jan-19-1)
+- LabVIEW 2018 FPGA Module
+- LabVIEW 2018 SP1 CompactRIO Module 18.5 (Jan, 2019)
+- LabVIEW 2018 FPGA Module Xilinx Compilation Tool for Vivado 2017.2
```

Comment:
I think this is optional if you use the NI cloud to compile the FPGA. Isn't it?

Reply:
You are right. Let me leave it with a comment (optional), because, if you don't want to use the NI cloud compile, it is good to know wich version to download.